### PR TITLE
Match player info and martial arts UI

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Interface "Match player info and martial arts UI."`

#### Purpose of change

**Last PR got a bit messy. New PRs will be smaller.**

This is just an idea for UI consistency as suggested in #12112.

Hope you like it.
<br />
#### Screenshot(s)

grafik

grafik

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -5636,6 +5639,23 @@ void player::on_effect_int_change( const efftype_id &eid, int intensity, body_pa
...
+    }
+
+    return *tdata;
+}
+
+void player::set_targeting_data( const targeting_data &td )
+{
+    tdata.reset( new targeting_data( td ) );
+}
+
```
Comment (Pupsi-Mupsi) [inline src/player.cpp]: I'm not sure how this got in my PR...

Comment (olanti-p) [reply src/player.cpp]: These functions and `#include "ranged.h"` were removed in #38605 if it helps

File: src/player_display.cpp [hunk 2]
```diff
@@ -1308,10 +1311,9 @@ void player::disp_info()
     ctxt.register_action( "HELP_KEYBINDINGS" );
     std::string action;
 
-    std::string help_msg = string_format( _( "Press %s for help." ),
-                                          ctxt.get_desc( "HELP_KEYBINDINGS" ) );
-    mvwprintz( w_tip, point( FULL_SCREEN_WIDTH - utf8_width( help_msg ), 0 ), c_light_red, help_msg );
-    help_msg.clear();
+    right_print( w_tip, 0, +4, c_white, string_format(
++                     _( "< [<color_yellow>%s</color>] keybindings >" ),
++                     ctxt.get_desc( "HELP_KEYBINDINGS" ) ) );
```
Comment (BevapDin) [inline src/player_display.cpp]:
```suggestion
                    ctxt.get_desc( "HELP_KEYBINDINGS" ) ) );
```
Or something alike, but the "+" should go away. Same on the line above.

Comment (Pupsi-Mupsi) [reply src/player_display.cpp]: Thanks a lot!

File: src/player.cpp (hunk 1 above)
Comment (Pupsi-Mupsi) [reply src/player.cpp]: Thank you!

**Pull request merged**